Fix implicit declaration of functions socket and connect and wrong
number of arguments for those funtions.
--- configure.orig	2014-01-18 06:34:00.000000000 -0600
+++ configure	2021-07-02 02:32:57.000000000 -0500
@@ -75,7 +75,8 @@
 EOF
 
 if ac_library_find_v 'socket and connect' "" "-lsocket -lnsl" <<EOF
-int main() { socket(); connect(); return 0; }
+#include <sys/socket.h>
+int main() { socket(0, 0, 0); connect(0, 0, 0); return 0; }
 EOF
 then :
 else
